packages/ui/src/theme/css/component/authenticator.scss

```diff
@@ -17,6 +17,16 @@
 
   /* Having a z-index at least "wins" by default */
   z-index: 1;
+
+  /* top-align at the top 20% of the screen */
+  padding-top: 20vh;
+  align-content: flex-start; // top align
+}
+
+[data-amplify-authenticator][data-variation='modal']
+  [data-amplify-router-content] {
+  max-height: 60vh;
```

Comment:
What if we make this full screen height on mobile, that way they don't need to scroll unnecessarily. Note that we need to beware of the `100vh` [issues on mobile ](https://css-tricks.com/css-fix-for-100vh-in-mobile-webkit/)if we do this.

Reply:
Done here! 06d96fca2d3e0a33e98de9c865ef14a0b49ac7de

Android:

![Screen Shot 2021-12-01 at 1 07 29 PM](https://user-images.githubusercontent.com/43682783/144317434-41145fa6-bfcb-4e24-9742-e68ca245e74c.png)

iOS (flex gap is a different problem, it's only [supported on safari 14.8+](https://caniuse.com/flexbox-gap)):

![image](https://user-images.githubusercontent.com/43682783/144317640-46559392-7957-454e-9710-48536626a05c.png)